[Android] App crashes when Ad consent is given and Bitdefender is installed

This one is a bit special, but I have several users, that are using Bitdefender on their Android devices. The issue does not exist, when Bitdefender is not installed. Possibly other security products may cause the same issue.
When my App starts the first time, the ad consent is not automatically given. When the setup wizard finishes, the default Google-Dialog appears and you have the choice to agree or deny the ads.

When the ads are denied, everything works fine. When you give the ad consent, the app crashes immediately. This also happens on each (re)start of the app after, as the ad consent is given now and seems to be loaded on app startup.

How to reproduce:
1. Have an app where users can give consent to ads or where this is done automatically. (if you need an example, try my one from the App store: Fahrplanalarm (it has a dual language interface ;))
2. Install Bitdefender, start the Bitdefender app the first time and finish the configuration. You don't need to book a plan. If you are at this step, you can select "later" at the top right corner. 
3. Start your app. It will crash

More details:
I tested it on a physical device, connected via USB. If you debug the app, it will throw the following exception in Visual Studio:
`System.NotImplementedException: 'The method or operation is not implemented.'`.
This is the call stack:
```
 	0x10 in Android.Runtime.RuntimeNativeMethods.monodroid_debugger_unhandled_exception	C#
 	0x2C in Android.Runtime.DynamicMethodNameCounter.7	C#
 	0x5 in Avalonia.Controls.Automation.Peers.InteropAutomationPeer.GetOrCreateChildrenCore	C#
 	0x1 in Avalonia.Automation.Peers.AutomationPeer.GetChildren	C#
 	0x12 in Avalonia.Android.AvaloniaAccessHelper.OnPopulateNodeForVirtualView	C#
 	0x12 in AndroidX.CustomView.Widget.ExploreByTouchHelper.n_OnPopulateNodeForVirtualView_ILandroidx_core_view_accessibility_AccessibilityNodeInfoCompat_ at C:\a\_work\1\s\generated\androidx.customview.customview\obj\Release\net8.0-android\generated\src\AndroidX.CustomView.Widget.ExploreByTouchHelper.cs:403,4	C#
 	0x1D in Android.Runtime.DynamicMethodNameCounter.7	C#
```

And this is the output (I shortend some lines with `[...]`):
```
[UserMessagingPlatform] Action[start_transparency_status_updates]: {}
[UserMessagingPlatform] Receive consent action: consent://consent/?action=configure_app_assets
[UserMessagingPlatform] Action[configure_app_assets]: {}
[UserMessagingPlatform] Wall html loaded.
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Receive consent action: consent://consent/?action=load_complete&args=%7B%22status%22%3A%22ok%22%7D
[UserMessagingPlatform] Action[load_complete]: {"status":"ok"}
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Receive consent action: consent://consent/?action=load_complete&args=%7B%22status%22%3A%22ok%22%7D
[UserMessagingPlatform] Action[load_complete]: {"status":"ok"}
[UserMessagingPlatform] Receive consent action: consent://consent/?action=start_transparency_status_updates
[UserMessagingPlatform] Action[start_transparency_status_updates]: {}
[UserMessagingPlatform] Receive consent action: consent://consent/?action=configure_app_assets
[UserMessagingPlatform] Action[configure_app_assets]: {}
[UserMessagingPlatform] Wall html loaded.
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Stored info not exists: IDFA_freqCapNumViews
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAnalyticsStoragePurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdPersonalizationPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: UMP_CoMoAdUserDataPurposeConsentStatus
[UserMessagingPlatform] Stored info not exists: IABTCF_idfaFlowControl
[UserMessagingPlatform] Stored info not exists: UMP_CoMoManagedByUmpSdk
[UserMessagingPlatform] Receive consent action: consent://consent/?action=load_complete&args=%7B%22status%22%3A%22ok%22%7D
[UserMessagingPlatform] Action[load_complete]: {"status":"ok"}
[UserMessagingPlatform] Receive consent action: consent://consent/?action=write&args=%7B%22IABTCF_CmpSdkID%22%3A300%2C%22IABTCF_CmpSdkVersion%22%3A2%2C%22IABTCF_PolicyVersion%22%3A5%2C%22IA […]
[UserMessagingPlatform] Action[write]: {"IABTCF_CmpSdkID":300,"IABTCF_CmpSdkVersion":2,"IABTCF_PolicyVersion":5,"IABTCF_gdprApplies":1,"IABTCF_PublisherCC":"DE","IABTCF_PurposeOneTreatment":0,"IABTCF_UseNonStandardStacks":0,"IABTCF_VendorConsents":"100000000111111100001011101100010001101100001000010100001010001000001110100111011100100000101000110000010000110000000010000100011100000000000100000100000001100010110000000011000000000000000001100000000100000011000000000000000000001100000100100001000000100000000001000000001010000010010000010001100000000100000001001000000000010100001000000000001000001000000000000000000000101000000001001000000100000000000000000010000000001000000000000000000000000001010000000000000000100000000000000001010100000000000000000100000000000000000011000000000000100000000100000000100000100000000000000000000010000000000000000000000000000000010000000000100000000000000000000000000000001000100000000000000000010001000000001000000000000000000000000000010000000000010000000000000010010000000010000000000000000000000000100000000001001000000000000000000000000000000000000000000000000000000000001000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000001000000000000000001000000000000000000000000000000000000000000000010001000000000000000000000000000000000000000000000000000000100000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000001","IABTCF_VendorLegitimateInterests":"100000000111111100001011101100010001101100001000010100001010001000001110000101011100100000101000110000010000110000000010000101011100000000000100000000000001100010110000000011000000000000000001100000000100000011000000000000000000001100000100100001000000100000000001000000001010010010010000010001001000000100000000001000000000010000001000000000000000001000000000000000000000100000000001000000000100000000000000000010100000001000000010000000000000000000010000000000000000000000000000000001010100000000000000000100000000000000000011000000000000100000000100000000100000100000010000000000000010000000000000000000000001000000010000000000000000000000000000000000000000001000100000000000000000010001000000001000000000000000000000000000010000000000010000000000000010000000000010000000000000000000000000100000000001001000000000000000000000000000000000000000000000000000000000001000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000011000000000000000000000000000000000000000000000010001000000000000000000000000000000000000000000000000000000100000000100000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","IABTCF_PurposeConsents":"11111111111","IABTCF_PurposeLegitimateInterests":"01000011111","IABTCF_SpecialFeaturesOptIns":"1","IABTCF_TCString":"CQX1pUAQX1pUAEsACBDEB8FoAP_gAEPgAAYgK1IB_C7EbCFCiDp3IKMEMAhHABBAYsAwAAYBAwAADBIQIAQCgkEYBASAFCACCAAAKASBAAAgCAAAAUAAIAAVAABAAAwAIBAIIAAAgAAAAEAIAAAACIAAEQCAAAAEAEAAkAgAAAIASAAAAAAAAACBAAAAAAAAAAAAAAAABAAAAQAAQAAAAAAAiAAAAAAAABAIAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAABAAAAAAAQR2QD-F2I2EKFEHCuQUYIYBCuACAAxYBgAAwCBgAAGCQgQAgFJIIkCAEAIEAAEAAAQAgCAABQEBAAAIAAAAAqAACAABgAQCAQQIABAAAAgIAAAAAAEQAAIgEAAAAIAIABABAAAAQAkAAAAAAAAAECAAAAAAAAAAAAAAAAAAAAAEABgAAAAAABEAAAAAAAACAQIAAA","IABTCF_AddtlConsent":"2~61.89.122.184.196.230.314.442.445.494.550.576.827.1029.1033.1046.1047.1051.1097.1126.1166.1301.1342.1415.1725.1765.1942.1958.1987.2068.2072.2074.2107.2213.2219.2223.2224.2328.2331.2387.2416.2501.2567.2568.2575.2657.2686.2778.2869.2878.2908.2920.2963.3005.3023.3100.3126.3219.3234.3235.3253.3309.3731.6931.8931.13731.15731.33931~dv.","IABTCF_UserConsentRecordId":"E38D9907
[UserMessagingPlatform] Receive consent action: consent://consent/?action=dismiss&args=%7B%22status%22%3A%22CONSENT_SIGNAL_SUFFICIENT%22%7D
[UserMessagingPlatform] Receive consent action: consent://consent/?action=complete
[UserMessagingPlatform] Writing to storage: [IABTCF_CmpSdkID] 300
[UserMessagingPlatform] Writing to storage: [IABTCF_CmpSdkVersion] 2
[UserMessagingPlatform] Writing to storage: [IABTCF_PolicyVersion] 5
[UserMessagingPlatform] Writing to storage: [IABTCF_gdprApplies] 1
[UserMessagingPlatform] Writing to storage: [IABTCF_PublisherCC] DE
[UserMessagingPlatform] Writing to storage: [IABTCF_PurposeOneTreatment] 0
[UserMessagingPlatform] Writing to storage: [IABTCF_UseNonStandardStacks] 0
[UserMessagingPlatform] Writing to storage: [IABTCF_VendorConsents] 100000[...]0001
[UserMessagingPlatform] Writing to storage: [IABTCF_VendorLegitimateInterests] 100000[...]000000
[UserMessagingPlatform] Writing to storage: [IABTCF_PurposeConsents] 11111111111
[UserMessagingPlatform] Writing to storage: [IABTCF_PurposeLegitimateInterests] 01000011111
[UserMessagingPlatform] Writing to storage: [IABTCF_SpecialFeaturesOptIns] 1
[UserMessagingPlatform] Writing to storage: [IABTCF_TCString] CQX1pUAQX1pUAE[...]QIAAA
[UserMessagingPlatform] Writing to storage: [IABTCF_AddtlConsent] 2~61[...]3931~dv.
[UserMessagingPlatform] Writing to storage: [IABTCF_UserConsentRecordId] E38[...]D9
[UserMessagingPlatform] Action[dismiss]: {"status":"CONSENT_SIGNAL_SUFFICIENT"}
[UserMessagingPlatform] Action[complete]: {}
[DynamiteModule] Local module descriptor class for com.google.android.gms.ads.dynamite not found.
[DynamitePackage] Instantiating com.google.android.gms.ads.ChimeraAdManagerCreatorImpl
[Ads] Use RequestConfiguration.Builder().setTestDeviceIds(Arrays.asList("1F7EA9766D4B7731E11EB1D100016ABB")) to get test ads on this device.
[rin.hns_xamari] Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.ads.eventattestation.internal.i>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/displayhash/DisplayHashResultCallback;
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.internal.l.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.nonagon.aw.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.internal.eventattestation.b.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.webview.n.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.z com.google.android.gms.ads.nonagon.render.aa.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:30)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.banner.ah.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.util.Map m84.dgl.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:41)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.f.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.cn com.google.android.gms.ads.nonagon.render.co.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:56)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.ag.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:38)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.ad.common.af com.google.android.gms.ads.nonagon.ad.banner.ag.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.U(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:493)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.ab.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.bf.bj(int, android.os.Parcel, android.os.Parcel) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:842)
[rin.hns_xamari]   at boolean m84.aoe.onTransact(int, android.os.Parcel, android.os.Parcel, int) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at boolean android.os.Binder.transact(int, android.os.Parcel, android.os.Parcel, int) (Binder.java:667)
[rin.hns_xamari]   at android.os.Parcel com.google.android.gms.internal.ads.zzaxb.zzcZ(int, android.os.Parcel) (com.google.android.gms:play-services-ads-base@@24.0.0:2)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.zzbv.zzab(com.google.android.gms.ads.internal.client.zzm) (com.google.android.gms:play-services-ads-lite@@24.0.0:3)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.client.zzek.zzm(com.google.android.gms.ads.internal.client.zzeh) (com.google.android.gms:play-services-ads-lite@@24.0.0:28)
[rin.hns_xamari]   at void com.google.android.gms.ads.zzg.run() (com.google.android.gms:play-services-ads-lite@@24.0.0:1)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.displayhash.DisplayHashResultCallback" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:134)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.internal.l.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.nonagon.aw.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.internal.eventattestation.b.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.webview.n.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.z com.google.android.gms.ads.nonagon.render.aa.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:30)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.banner.ah.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.util.Map m84.dgl.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:41)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.f.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.cn com.google.android.gms.ads.nonagon.render.co.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:56)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.ag.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:38)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.ad.common.af com.google.android.gms.ads.nonagon.ad.banner.ag.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.U(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:493)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.ab.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.bf.bj(int, android.os.Parcel, android.os.Parcel) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:842)
[rin.hns_xamari]   at boolean m84.aoe.onTransact(int, android.os.Parcel, android.os.Parcel, int) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at boolean android.os.Binder.transact(int, android.os.Parcel, android.os.Parcel, int) (Binder.java:667)
[rin.hns_xamari]   at android.os.Parcel com.google.android.gms.internal.ads.zzaxb.zzcZ(int, android.os.Parcel) (com.google.android.gms:play-services-ads-base@@24.0.0:2)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.zzbv.zzab(com.google.android.gms.ads.internal.client.zzm) (com.google.android.gms:play-services-ads-lite@@24.0.0:3)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.client.zzek.zzm(com.google.android.gms.ads.internal.client.zzeh) (com.google.android.gms:play-services-ads-lite@@24.0.0:28)
[rin.hns_xamari]   at void com.google.android.gms.ads.zzg.run() (com.google.android.gms:play-services-ads-lite@@24.0.0:1)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] 
[rin.hns_xamari] Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.ads.eventattestation.internal.i>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/displayhash/DisplayHashResultCallback;
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.internal.l.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.nonagon.aw.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.internal.eventattestation.b.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.webview.n.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.z com.google.android.gms.ads.nonagon.render.aa.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:30)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.banner.ah.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.util.Map m84.dgl.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:41)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.f.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.cn com.google.android.gms.ads.nonagon.render.co.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:56)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.ag.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:38)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.ad.common.af com.google.android.gms.ads.nonagon.ad.banner.ag.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.U(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:493)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.ab.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.bf.bj(int, android.os.Parcel, android.os.Parcel) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:842)
[rin.hns_xamari]   at boolean m84.aoe.onTransact(int, android.os.Parcel, android.os.Parcel, int) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at boolean android.os.Binder.transact(int, android.os.Parcel, android.os.Parcel, int) (Binder.java:667)
[rin.hns_xamari]   at android.os.Parcel com.google.android.gms.internal.ads.zzaxb.zzcZ(int, android.os.Parcel) (com.google.android.gms:play-services-ads-base@@24.0.0:2)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.zzbv.zzab(com.google.android.gms.ads.internal.client.zzm) (com.google.android.gms:play-services-ads-lite@@24.0.0:3)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.client.zzek.zzm(com.google.android.gms.ads.internal.client.zzeh) (com.google.android.gms:play-services-ads-lite@@24.0.0:28)
[rin.hns_xamari]   at void com.google.android.gms.ads.zzg.run() (com.google.android.gms:play-services-ads-lite@@24.0.0:1)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.displayhash.DisplayHashResultCallback" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:134)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.internal.l.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at com.google.android.gms.ads.internal.eventattestation.c com.google.android.gms.ads.nonagon.aw.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.internal.eventattestation.b.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.webview.n.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.z com.google.android.gms.ads.nonagon.render.aa.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:30)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.banner.ah.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.util.Map m84.dgl.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:41)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.f.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.render.cn com.google.android.gms.ads.nonagon.render.co.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:56)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.ad.common.ag.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:38)
[rin.hns_xamari]   at java.lang.Object m84.dgh.d() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:12)
[rin.hns_xamari]   at java.lang.Object m84.dgh.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at com.google.android.gms.ads.nonagon.ad.common.af com.google.android.gms.ads.nonagon.ad.banner.ag.b() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.U(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:493)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.f.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.nonagon.shim.ab.M(com.google.android.gms.ads.internal.client.k) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.bf.bj(int, android.os.Parcel, android.os.Parcel) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:842)
[rin.hns_xamari]   at boolean m84.aoe.onTransact(int, android.os.Parcel, android.os.Parcel, int) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:21)
[rin.hns_xamari]   at boolean android.os.Binder.transact(int, android.os.Parcel, android.os.Parcel, int) (Binder.java:667)
[rin.hns_xamari]   at android.os.Parcel com.google.android.gms.internal.ads.zzaxb.zzcZ(int, android.os.Parcel) (com.google.android.gms:play-services-ads-base@@24.0.0:2)
[rin.hns_xamari]   at boolean com.google.android.gms.ads.internal.client.zzbv.zzab(com.google.android.gms.ads.internal.client.zzm) (com.google.android.gms:play-services-ads-lite@@24.0.0:3)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.client.zzek.zzm(com.google.android.gms.ads.internal.client.zzeh) (com.google.android.gms:play-services-ads-lite@@24.0.0:28)
[rin.hns_xamari]   at void com.google.android.gms.ads.zzg.run() (com.google.android.gms:play-services-ads-lite@@24.0.0:1)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] 
[DynamiteModule] Local module descriptor class for com.google.android.gms.ads.dynamite not found.
[rin.hns_xamari] Accessing hidden field Landroid/graphics/Insets;->NONE:Landroid/graphics/Insets; (light greylist, linking)
[Ads] Invoke Firebase method getInstance error.
[Ads] The Google Mobile Ads SDK will not integrate with Firebase. Admob/Firebase integration requires the latest Firebase SDK jar, but Firebase SDK is either missing or out of date
[AudioManager] In isSpeakerphoneOn(), calling application: hns_xamarin.hns_xamarin
[ServiceBindIntentUtils] Dynamic lookup for intent failed for action: com.google.android.gms.ads.service.CACHE
[rin.hns_xamari] Accessing hidden field Landroid/graphics/Insets;->top:I (light greylist, linking)
[rin.hns_xamari] Accessing hidden field Landroid/graphics/Insets;->bottom:I (light greylist, linking)
[rin.hns_xamari] Accessing hidden field Landroid/graphics/Insets;->left:I (light greylist, linking)
[rin.hns_xamari] Accessing hidden field Landroid/graphics/Insets;->right:I (light greylist, linking)
[rin.hns_xamari] Explicit concurrent copying GC freed 42506(3MB) AllocSpace objects, 28(7MB) LOS objects, 49% free, 8MB/17MB, paused 64us total 42.766ms
[rin.hns_xamari] The ClassLoaderContext is a special shared library.
[rin.hns_xamari] Rejecting re-init on previously-failed class java.lang.Class<m84.alh>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/content/pm/PackageManager$OnChecksumsReadyListener;
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at void m84.amn.u(android.content.Context, m84.amm) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:144)
[rin.hns_xamari]   at m84.amn m84.amn.v(android.content.Context, m84.amm) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:1)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.k.r(boolean) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:78)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.k.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:51)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] Caused by: java.lang.ClassNotFoundException: Didn't find class "android.content.pm.PackageManager$OnChecksumsReadyListener" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:134)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at void m84.amn.u(android.content.Context, m84.amm) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:144)
[rin.hns_xamari]   at m84.amn m84.amn.v(android.content.Context, m84.amm) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:1)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.k.r(boolean) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:78)
[rin.hns_xamari]   at void com.google.android.gms.ads.internal.k.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:51)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] 
[rin.hns_xamari] Rejecting re-init on previously-failed class java.lang.Class<com.google.android.ads.tasks.appdigest.a>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/content/pm/PackageManager$OnChecksumsReadyListener;
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at m84.dbh m84.amn.m(android.content.Context, com.google.ads.afma.g) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:128)
[rin.hns_xamari]   at java.lang.String m84.amh.t(android.content.Context, java.lang.String, int, android.view.View, android.app.Activity, byte[]) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:108)
[rin.hns_xamari]   at java.lang.String m84.amh.g(android.content.Context) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:14)
[rin.hns_xamari]   at java.lang.String com.google.android.gms.ads.internal.k.h(android.content.Context, byte[]) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:20)
[rin.hns_xamari]   at java.lang.String com.google.android.gms.ads.internal.k.g(android.content.Context) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:2)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.load.l.call() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object m84.clm.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at void m84.ckt.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:19)
[rin.hns_xamari]   at void m84.cln.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] Caused by: java.lang.ClassNotFoundException: Didn't find class "android.content.pm.PackageManager$OnChecksumsReadyListener" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/000004f0/dl-AdsFdrDynamite.integ_251815204100000.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:134)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClassNative(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile) (DexFile.java:-2)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.defineClass(java.lang.String, java.lang.ClassLoader, java.lang.Object, dalvik.system.DexFile, java.util.List) (DexFile.java:283)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexFile.loadClassBinaryName(java.lang.String, java.lang.ClassLoader, java.util.List) (DexFile.java:276)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList$Element.findClass(java.lang.String, java.lang.ClassLoader, java.util.List) (DexPathList.java:724)
[rin.hns_xamari]   at java.lang.Class dalvik.system.DexPathList.findClass(java.lang.String, java.util.List) (DexPathList.java:486)
[rin.hns_xamari]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:132)
[rin.hns_xamari]   at java.lang.Class m2.as.loadClass(java.lang.String, boolean) (:com.google.android.gms.dynamite_dynamiteloader@253534013@25.35.34 (100400-0):21)
[rin.hns_xamari]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[rin.hns_xamari]   at m84.dbh m84.amn.m(android.content.Context, com.google.ads.afma.g) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:128)
[rin.hns_xamari]   at java.lang.String m84.amh.t(android.content.Context, java.lang.String, int, android.view.View, android.app.Activity, byte[]) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:108)
[rin.hns_xamari]   at java.lang.String m84.amh.g(android.content.Context) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:14)
[rin.hns_xamari]   at java.lang.String com.google.android.gms.ads.internal.k.h(android.content.Context, byte[]) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:20)
[rin.hns_xamari]   at java.lang.String com.google.android.gms.ads.internal.k.g(android.content.Context) (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:2)
[rin.hns_xamari]   at java.lang.Object com.google.android.gms.ads.nonagon.load.l.call() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:7)
[rin.hns_xamari]   at java.lang.Object m84.clm.a() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:3)
[rin.hns_xamari]   at void m84.ckt.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:19)
[rin.hns_xamari]   at void m84.cln.run() (:com.google.android.gms.policy_ads_fdr_dynamite@251815214@251815204017.785561508.785561508:5)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1167)
[rin.hns_xamari]   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:641)
[rin.hns_xamari]   at void java.lang.Thread.run() (Thread.java:764)
[rin.hns_xamari] 
[rin.hns_xamari] Explicit concurrent copying GC freed 9454(838KB) AllocSpace objects, 8(416KB) LOS objects, 49% free, 8MB/16MB, paused 59us total 34.327ms
[Choreographer] Skipped 73 frames!  The application may be doing too much work on its main thread.
[OpenGLRenderer] Davey! duration=1231ms; Flags=0, IntendedVsync=29019810156721, Vsync=29021026823339, OldestInputEvent=9223372036854775807, NewestInputEvent=0, HandleInputStart=29021031440334, AnimationStart=29021031488563, PerformTraversalsStart=29021032404292, DrawStart=29021038484448, SyncQueued=29021039225698, SyncStart=29021039334032, IssueDrawCommandsStart=29021039496115, SwapBuffers=29021040559240, FrameCompleted=29021041337261, DequeueBufferDuration=312000, QueueBufferDuration=410000, 
[ServiceBindIntentUtils] Dynamic lookup for intent failed for action: com.google.android.gms.ads.service.START
[rin.hns_xamari] The ClassLoaderContext is a special shared library.
**System.NotImplementedException:** 'The method or operation is not implemented.'
[libc] Requested dump for tid 12777 (rin.hns_xamarin)
```

--- a/src/Jc.AdMob.Avalonia/AdMobAutomationPeer.cs
+++ b/src/Jc.AdMob.Avalonia/AdMobAutomationPeer.cs
@@ -1,0 +1,14 @@
+using System.Collections.Generic;
+using Avalonia.Automation.Peers;
+using Avalonia.Controls;
+
+namespace Jc.AdMob.Avalonia;
+
+internal sealed class AdMobAutomationPeer(Control owner) : ControlAutomationPeer(owner)
+{
+    protected override IReadOnlyList<AutomationPeer> GetOrCreateChildrenCore()
+        => [];
+
+    protected override AutomationControlType GetAutomationControlTypeCore()
+        => AutomationControlType.Custom;
+}

--- a/src/Jc.AdMob.Avalonia/BannerAd.cs
+++ b/src/Jc.AdMob.Avalonia/BannerAd.cs
@@ -1,4 +1,5 @@
 ﻿using System;
+using Avalonia.Automation.Peers;
 using Avalonia.Controls;
 using Avalonia.Platform;
 
@@ -29,7 +30,10 @@ public sealed class BannerAd : NativeControlHost
         return Implementation?.CreateControl(UnitId, this, parent, () => base.CreateNativeControlCore(parent))
                ?? base.CreateNativeControlCore(parent);
     }
-    
+
+    protected override AutomationPeer OnCreateAutomationPeer()
+        => new AdMobAutomationPeer(this);
+
     internal void AdLoaded(object? sender, EventArgs e) => OnAdLoaded?.Invoke(sender, e);
     internal void AdFailedToLoad(object? sender, AdError e) => OnAdFailedToLoad?.Invoke(sender, e);
     internal void AdImpression(object? sender, EventArgs e) => OnAdImpression?.Invoke(sender, e);

